### ISSUE TITLE
Librarian/staff can view renewal requests for B&C tools

### ISSUE THREAD

**Issue by rkim00** [rubyforgood/circulate#307]
This story probably comes after #123 .

As a Librarian, I want to be able to view renewal requests submitted by members for B&C tools (#123 ) , so that I can approve (#306 ) or deny (#305 ) the request.

Display renewal requests on a new page for Librarians. See mock-ups (https://drive.google.com/file/d/1sbf62tG2x7nUCHX5DN8_neipsGQqtdqs/view?usp=sharing , bottom of page).

Librarians should be able to toggle between all, pending ("requested" on mock-up), and closed renewal requests.

Not pictured in mock-up:
- name of tool should link to the Librarian detail on that tool (e.g., https://circulate-staging.herokuapp.com/admin/items/1167)
- name of member should link to Member detail on that Member (e.g., https://circulate-staging.herokuapp.com/admin/members/585)
- would be nice to have a column next to Name called "Number" that shows the inventory number of the item (e.g., B-1234). However, this isn't necessarily for the first version of this story.

Note that "holds" below the item Name should be the number of holds on that Item, with nothing displaying if there are no holds. This could also be a separate column if that makes implementation easier.

**Comment by pjsier** [comment]
I'm interested in taking this one on as a follow up to my last PR!

**Comment by jim** [comment]
Closed by #456
